pkg/git/git_test.go

```diff
@@ -8,14 +8,14 @@ func TestRemoteURL(t *testing.T) {
 	t.Parallel()
 
 	actual := RemoteURL("example.com", "app")
-	expected := "ssh://git@example.com:2222/app.git"
+	expected := "ssh://git@example-builder.com:2222/app.git"
```

Comment:
this doesn't seem right to me. So if someone wanted to expose the builder on example.com, they'd have to own both example.com and example-builder.com?